src/shared/stores/entry-form-store.js

```diff
@@ -4,6 +4,7 @@ import create from 'zustand'
 const inititalState = {
     errors: {},
     warnings: {},
+    completeAttempted: false,
```

Comment:
I couldn't totally decide where to put this, but thought that since it was going to apply error styling to cells, I would put it in the store with the errors information.